### PR TITLE
Fixes for WordPress on App Engine standard.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
Jenkins passes too :)

File: appengine/wordpress/src/WordPressSetup.php [hunk 1]
```diff
@@ -249,14 +270,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-            'db_name' => 'wp',
-            'db_user' => 'wp',
-            'db_password' => ''
-        );
         if ($env === self::STANDARD_ENV) {
+            $keys = array(
+                'project_id' => '',
+                'db_instance' => 'wp',
+                'db_name' => 'wp',
+                'cloudsql_root_password' => '',
```
Comment (bshaffer) [inline appengine/wordpress/src/WordPressSetup.php]: shouldn't this just be `cloudsql_password`? I don't think the user being root should be a requirement.

**Comment by bshaffer** [comment]
Looks good, just one suggestion.

File: appengine/wordpress/src/WordPressSetup.php (hunk 1 above)
Comment (tmatsuo) [inline appengine/wordpress/src/WordPressSetup.php]: This is a bit tricky, but from App Engine standard, it always uses root@localhost.

Comment (tmatsuo) [inline appengine/wordpress/src/WordPressSetup.php]: Ah, sorry you are right. You can use other user too.

Comment (tmatsuo) [inline appengine/wordpress/src/WordPressSetup.php]: I have changed it to allow other db user. Also always ask you to use the same password locally.

**Comment by tmatsuo** [comment]
@bshaffer PTAL

**Comment by bshaffer** [comment]
LGTM sir

**Pull request merged**